src/ai/backend/manager/registry.py

```diff
@@ -777,6 +777,10 @@ async def enqueue_session(
 
         mounts = kernel_enqueue_configs[0]['creation_config'].get('mounts') or []
         mount_map = kernel_enqueue_configs[0]['creation_config'].get('mount_map') or {}
+        root_mounts = list(set(map(lambda x: x.split("/")[0], mounts)))
+        root_mount_map = dict()
```

Comment:
For intrinsict container types, it's recommended to use the native syntax (`{}` instead of `dict()`).

Though, using `dict()` may be better if there are set expressions around it to avoid confusion of that `{}` is an empty dict while `{something, ...}` (without `:`) is a set expression.  I think this case is not.